bridge-cli/src/fast_bridge_command.rs

```diff
@@ -0,0 +1,151 @@
+use crate::{combined_config, CliConfig, Network};
+use clap::Subcommand;
+use ethers_core::types::{Address, TxHash};
+use fast_bridge::{FastBridge, FastBridgeBuilder};
+use near_primitives::types::AccountId;
+use std::str::FromStr;
+
+#[derive(Subcommand, Debug)]
+pub enum FastBridgeSubCommand {
+    Transfer {
+        #[clap(short, long)]
+        token: String,
+        #[clap(short, long)]
+        amount: u128,
+        #[clap(short, long)]
+        fee: u128,
+        #[clap(short, long)]
+        eth_token_address: String,
```

Comment:
It is better to extract this information automatically and pass only one of them, either the token on NEAR or on Ethereum.